regression/cbmc/r_w_ok_inconsistent_dead/test-global-no-cp.desc

```diff
@@ -0,0 +1,10 @@
+FUTURE
```

Comment:
What does FUTURE mean?

Reply:
Tests that currently don't pass but are not bugs are marked as FUTURE. This is typically used for tests for unimplemented features. Here the test doesn't pass due to the overapproximate behaviour of `__CPROVER_r_ok()`. If it will be made more pecise in the future it should pass.